pkg/watcher/resource-event-handler/pod_test.go

```diff
@@ -18,20 +18,26 @@ var _ = Describe("Pod Event Handler", func() {
 	Context("OnAdd", func() {
 		It("On add pod event", func() {
 			pod1 := &kapi.Pod{ObjectMeta: metav1.ObjectMeta{Annotations: map[string]string{
-				v1.NetworkAttachmentAnnot: `[{"name":"test"},{"name":"test2", "cni-args":{"mellanox.infiniband.app":"configured"}}]`}},
+				v1.NetworkAttachmentAnnot: `[{"name":"test", "namespace":"default"},{"name":"test2", "cni-args":{"mellanox.infiniband.app":"configured"}}]`}},
 				Spec: kapi.PodSpec{NodeName: "test"}}
 			pod2 := &kapi.Pod{ObjectMeta: metav1.ObjectMeta{Annotations: map[string]string{
-				v1.NetworkAttachmentAnnot: `[{"name":"test"}]`}},
+				v1.NetworkAttachmentAnnot: `[{"name":"test", "namespace":"default"}]`}},
+				Spec: kapi.PodSpec{NodeName: "test"}}
+			pod3 := &kapi.Pod{ObjectMeta: metav1.ObjectMeta{Annotations: map[string]string{
+				v1.NetworkAttachmentAnnot: `[{"name":"test", "namespace":"test"}]`}},
```

Comment:
can we have other name other then test for the namespace its the same as the network name